### ISSUE TITLE
focus in the folder view jumps to the selected folder

### ISSUE THREAD

**Issue by Rude-Boy** [RealRaven2000/QuickFolders#259]
When I open a folder from the QuickFolders bar, the focus in the folder view jumps to that folder.
Is there any way to prevent the folder view from changing?

**Comment by RealRaven2000** [comment]
> When I open a folder from the QuickFolders bar, the focus in the folder view jumps to that folder. Is there any way to prevent the folder view from changing?

no, because QuickFolders "Jumps to the folder", and I do not want to disconnect the folder tree from that. Why do you want do this? 

Remember the best way to use the folder tree is "look, don't touch". Stop dragging your emails there, stop using it for navigation. This is what QuickFolders (and it's quickJump button) is for. Once you start learning this way of navigation it will save you hours of scrolling every week.

**Comment by Rude-Boy** [comment]
I used the folder tree a lot before I installed QuickFolders a week ago. It is still hard for me to work without the folder tree at the moment. But thanks for the advice. I am very happy with QuickFolders.

**Comment by RealRaven2000** [comment]
> I used the folder tree a lot before I installed QuickFolders a week ago. It is still hard for me to work without the folder tree at the moment. But thanks for the advice. I am very happy with QuickFolders.

Sure thing - have a look at my video on "how to skip the abyss" for some advanced concepts:
https://youtu.be/NxoU95aUw_8?t=99

It shows the Navigation toolbar and  explains the additional "skip [unread] folder" and also quickJump / quickMove buttons. This feature has since been vastly improved. Once you click it (or use the Shortcut Shift+M if you have a Pro License) you can see a help panel:
![image](https://user-images.githubusercontent.com/6314674/164686367-2c9d1b64-2c8d-49f9-82d8-ab43079930c1.png)

![image](https://user-images.githubusercontent.com/6314674/164686871-b37636be-b2dc-4c66-ad14-b732053c9f0a.png)

If you strategically add the the correct parent folders to the QuickFolders toolbar as tabs this will already give you a lot of access (and visibility) to you most important incoming mails; quickJump just completes the picture to give you access to everything - I highly recommend trying out the `/`  and `>`  keys to look for child folders by entering either direct or any parent folders:
![image](https://user-images.githubusercontent.com/6314674/164687766-a21c493e-dfe0-413f-b743-46e1613194ef.png)

This alternative way of navigation takes a little time to get used to, but the folder tree is an awful "time sink" when you have many folders.
